pkg/platform/runtime/setup/implementations/installer/runtime.go

```diff
@@ -0,0 +1,32 @@
+package installer
+
+import (
+	"github.com/ActiveState/cli/pkg/platform/api/headchef/headchef_models"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+	"github.com/ActiveState/cli/pkg/platform/runtime/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/store"
+)
+
+type Setup struct {
```

Comment:
Can we add some comments here explaining the nature of this implementation and the fact that it is temporary. Add a link to https://activestatef.atlassian.net/browse/DX-846